components/ForceVector.vue

```diff
@@ -1,21 +1,27 @@
 <template>
-    <v-group>
-        <v-arrow :config="arrowConfig"/>
-        <v-circle 
-            :x="arrowConfig.points[2]" 
-            :y="arrowConfig.points[3]" 
-            :radius="8"
-            :fill="'red'"
-            :draggable="true"/>
-        <v-arrow v-if="props.showComponents" :config="xComponentConfig"/>
-        <v-arrow v-if="props.showComponents" :config="yComponentConfig"/>
-    </v-group>
+    <v-arrow :config="arrowConfig"/>
+    <v-circle 
+        :x="arrowConfig.points[2]" 
```

Comment:
Since we're early in this, do you think we should refactor and use our point object or a similar simplified point, and then have arrowConfig have a head and a tail as a point? Then we could say things like `arrowConfig.head.x` which would be very clear...

Reply:
Oh, nevermind, you can't do this because arrowConfig is defined by konva...

Reply:
Yeah, I _hate_ the array thing that Konva has for arrows. I imagine we could define some kind of wrapper at some point if we wanted though.